Core/src/BareFatJets.cpp

```diff
@@ -3,24 +3,54 @@
 
 
 BareFatJets::BareFatJets(): BareP4(){
-    p4 = NULL;
-    rawPt = NULL;
-    flavour = NULL;
-    tau1 = NULL;
-    tau2 = NULL;
-    tau3 = NULL;
-    trimmedMass = NULL;
-    prunedMass = NULL;
-    filteredMass = NULL;
-    softdropMass = NULL;
-    ak8_subjet = NULL;
-    ak8jet_hasSubjet = NULL;
-    ak8subjet_btag = NULL;
-    hbb = NULL;
-
 }
 
```

Comment:
Il 06.09.15 23:03, Yutaro Iiyama ha scritto:
>
> In Core/src/BareFatJets.cpp
> <https://github.com/MiT-HEP/NeroProducer/pull/106#discussion_r38826111>:
>
> > @@ -3,24 +3,54 @@
> >  
> >  
> >  BareFatJets::BareFatJets(): BareP4(){
> > -    p4 = NULL;
> > -    rawPt = NULL;
> > -    flavour = NULL;
> > -    tau1 = NULL;
> > -    tau2 = NULL;
> > -    tau3 = NULL;
> > -    trimmedMass = NULL;
> > -    prunedMass = NULL;
> > -    filteredMass = NULL;
> > -    softdropMass = NULL;
> > -    ak8_subjet = NULL;
> > -    ak8jet_hasSubjet = NULL;
> > -    ak8subjet_btag = NULL;
> > -    hbb = NULL;
> > -
> >  }
> >  
>
> No, delete will do nothing if the pointer is properly zero
> initialized. The initialization is done in the .h file.
>
Is this a new in C++11 ... awsome!


Andrea
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MiT-HEP/NeroProducer/pull/106/files#r38826111>.
>


-- 
●●●●●●●●●●●●●●●●●●●●●●●●●●
Andrea Carlo Marini, 
MIT Post-Doc,
c/o 32-4-A20 CERN